src/validation.cpp

```diff
@@ -3219,7 +3219,7 @@ static bool FindBlockPos(FlatFilePos &pos, unsigned int nAddSize, unsigned int n
 
     if ((int)nFile != nLastBlockFile) {
         if (!fKnown) {
-            LogPrintf("Leaving block file %i: %s\n", nLastBlockFile, vinfoBlockFile[nLastBlockFile].ToString());
+            LogPrint(BCLog::VALIDATION, "Leaving block file %i: %s\n", nLastBlockFile, vinfoBlockFile[nLastBlockFile].ToString());
```

Comment:
Thanks for making the change, but shouldn't this be in BLOCKSTORE too?
(or move the other to validation, I don't know, it seems that it would make sense to have the messages about block files in the same category)

Reply:
Good point. Moved both to `VALIDATION`.

Felt a bit bad to introduce `BLOCKSTORE` for a single log message: nice to get rid of it! :)